srcnn/competition_metric.py

```diff
@@ -0,0 +1,164 @@
+import numpy as np
```

Comment:
This file is referenced from aipos's github open source. @congruihuang shall we put it here or ref it?